redisinsight/api/config/staging.ts

```diff
@@ -1,6 +1,6 @@
 import { join } from 'path';
 
-const homedir = join(require('os').homedir(), process.env.APP_FOLDER_NAME || '.redisinsight-v2.0-stage');
+const homedir = join(require('os').homedir(), '.redisinsight-v2.0-stage');
```

Comment:
I'm afraid that integration tests on docker images will fail then. 
Why we need such changes?

Reply:
Because at the moment the stage and prod builds both refer to the same folder (.redisinsight-preview) inside homedir.  Or we can add the additional env variable for stage (APP_STAGE_FOLDER_NAME)

Reply:
> 

We should handle it elsewhere. 
For example: 
- in the circleci.yml (like it is done for autoupdate)
- or we can remove environment variable from the circleci and change defaults for each config file (dev, stage, prod)

Reply:
For now, I just updated the default homedir for production env

Reply:
It is not enough to change default value for production only. We need to remove circleci  APP_FOLDER_NAME env then

Reply:
Never mind. I'll remove it myself